### PR TITLE
Update to v1.0.4

### PR DESCRIPTION
**Pull request by synapticarbors**
@cshaley - I re-rendered the feedstock to get python 3.6 and updated to version 1.0.4. Hopefully this will help your other issues.

### CODE REVIEW

**Comment by synapticarbors** [comment]
Had to include `pytz` as a test requirement since it looks like the `django-feedstock` is missing it as a proper run time requirement. 

https://github.com/conda-forge/django-feedstock/issues/26

**Comment by ocefpaf** [comment]
Why not a PR to the Django feedstock instead  ;-)

**Comment by cshaley** [comment]
Looks like this is ready to merge!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,6 +26,10 @@ requirements:
     - djangorestframework
 
 test:
+  requires:
+    # Hack since the django feedstock appears not to have this correctly listed a run req.
+    - pytz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: This should be removed first.

**Comment by cshaley** [comment]
Is this ready to merge now?

**Pull request merged**